packages/babel-preset-env/test/use-built-ins-plugin.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+const babel = require("@babel/core");
+const assert = require("assert");
+const sinon = require("sinon");
+
+describe("use-built-ins-plugin", () => {
+  describe("usage option", () => {
+    it("should remove all babel-polyfill imports", () => {
+      const codeWithImports = `import "@babel/polyfill";
+import "@babel/polyfill";
+require("@babel/polyfill");
+require("@babel/polyfill");
+`;
```

Comment:
Can we make this look nice, since whitespaces are not important for this test? :slightly_smiling_face: 
```js
      const codeWithImports = `
        import "@babel/polyfill";
        import "@babel/polyfill";
        require("@babel/polyfill");
        require("@babel/polyfill");
      `;

```

Reply:
Sure, we can just `trim()` it 🙂